src/dosarray_run_http_experiment.sh

```diff
@@ -117,12 +117,19 @@ fi
 #      $ATTACK_LASTS_FOR.
 [ -z "${ATTACK_LASTS_FOR}" ] && ATTACK_LASTS_FOR=20
 
-# FIXME check that these values are sensible wrt each other -- e.g., that an attack doesn't last longer than the experiment.
+
 echo "EXPERIMENT_DURATION=${EXPERIMENT_DURATION}"
 echo "DOSARRAY_VIRT_INSTANCES=${DOSARRAY_VIRT_INSTANCES}"
 echo "ATTACK_STARTS_AT=${ATTACK_STARTS_AT}"
 echo "ATTACK_LASTS_FOR=${ATTACK_LASTS_FOR}"
 
+# check that an attack doesn't last longer than the experiment.
+if [ $(( ${ATTACK_STARTS_AT} + ${ATTACK_LASTS_FOR} )) -gt ${EXPERIMENT_DURATION} ]
```

Comment:
To provide more accurate message, could also check whether ATTACK_STARTS_AT is itself than EXPERIMENT_DURATION; this would exonerate ATTACK_LASTS_FOR.

Reply:
Done